### PR TITLE
KOGITO-4161: Doc - Create Management/Task Console quickstart sample project

### PR DESCRIPTION
**Pull request by pefernan**
There we go! some quickstarts work for the consoles... 

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: kogito-consoles-quickstarts/01-kogito-basic-example/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# 01.- Kogito Basic Example
+
+## Description
+
+This Quickstart showcases a basic implementation of the **Hiring** process. 
+
+For simplicity, this example doesn't include any authentication mechanism, so all tasks in will be assigned to a default user `john`.
+
+The required *Kogito and Infrastructure Services* for this example is:
```
Comment (sterobin) [inline kogito-consoles-quickstarts/01-kogito-basic-example/README.md]: "for this example are:"

**Review by sterobin — APPROVED** [review]
@pefernan Just a tiny comment, otherwise seems fine to me. Once we know that this will be in 1.2, I can throw in a quick reference in the task console docs section pointing to the relevant quickstarts.

File: kogito-consoles-quickstarts/01-kogito-basic-example/README.md (hunk 1 above)
Comment (pefernan) [reply kogito-consoles-quickstarts/01-kogito-basic-example/README.md]: ouch! good catch @sterobin

**Comment by nmirasch** [comment]
@pefernan I miss here some instructions for native running but regarding it's a quick start, maybe this simplifies the running. 
We will improve that with an index.html with more info to avoid the error accessing to the runtime service and a good conf to see the diagrams at management console.This works fine and as starting point it looks good to me. 
About the naming.. I see a bit weird the root folder name, IMO makes sense something more generic like kogito-quickstarts, wdyt?

**Comment by pefernan** [comment]
@nmirasch I purposely skipped the native build since it's already explained in the docs. About the naming... I can change it, I was focuins this as a quickstarts to consoles, but np

**Review by nmirasch — APPROVED** [review]

File: kogito-quickstarts/01-kogito-basic-example/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   depends_on:
+     - data-index
+
+  task-console:
+    container_name: task-console
+    image: quay.io/kiegroup/kogito-task-console-nightly:${KOGITO_VERSION}
+    ports:
+      - 8380:8080
+    depends_on:
+      - data-index
```
Comment (Sara4994) [inline kogito-quickstarts/01-kogito-basic-example/docker-compose/docker-compose.yml]: @pefernan how about adding the same condition here as well?

File: kogito-quickstarts/01-kogito-basic-example/docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      KAFKA_BOOTSTRAP_SERVERS: kafka:29092
+      KOGITO_DATA_INDEX_PROPS: -Dkogito.protobuf.folder=/home/kogito/data/protobufs/
+
+  management-console:
+   container_name: management-console
+   image: quay.io/kiegroup/kogito-management-console-nightly:${KOGITO_VERSION}
+   ports:
+     - 8280:8080
+   depends_on:
+     - data-index
```
Comment (Sara4994) [inline kogito-quickstarts/01-kogito-basic-example/docker-compose/docker-compose.yml]: @pefernan Can we add a condition to watch for data-index, if it has started and then start the console(condition: service_started). wdyt?

Comment (pefernan) [reply kogito-quickstarts/01-kogito-basic-example/docker-compose/docker-compose.yml]: @Sara4994 done!

**Comment by Sara4994** [comment]
@pefernan Looks good to me now. Thanks :)

**Review by Sara4994 — APPROVED** [review]

**Comment by pefernan** [comment]
jenkins retest this please

**Comment by pefernan** [comment]
jenkins retest this please

**Review by AjayJagan — APPROVED** [review]
thanks @pefernan :+1:

**Pull request merged**